tests/integration/test_algorithms.py

```diff
@@ -35,8 +35,8 @@ def test_teleportation_fidelity(setup_eng, pure):
         BS | (q[1], q[2])
 
         BS | (q[0], q[1])
-        MeasureHomodyne(0, select=0) | q[0]
-        MeasureHomodyne(np.pi / 2, select=0) | q[1]
+        MeasureHomodyne(0, select=0.07) | q[0]
+        MeasureHomodyne(np.pi / 2, select=0.1) | q[1]
```

Comment:
How come this was changed?

Reply:
If the postselected value is zero, scaling errors do not show up.

Reply:
:+1:

Reply:
Wait no, that's wrong. At the moment `test_teleportation_fidelity` does not use the measurement results to displace the output mode, so if the postselected value is nonzero the result is wrong. I'll fix it.

Reply:
Fixed, I added the required displacements.